src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.String.cs

```diff
@@ -391,28 +394,54 @@ private void WriteStringHelper(ReadOnlySpan<byte> utf8PropertyName, ReadOnlySpan
         /// <exception cref="ArgumentException">
         /// Thrown when the specified property name is too large.
         /// </exception>
+        /// <exception cref="ArgumentNullException">
+        /// The <paramref name="propertyName"/> parameter is <see langword="null"/>.
+        /// </exception>
         /// <exception cref="InvalidOperationException">
         /// Thrown if this would result in an invalid JSON to be written (while validation is enabled).
         /// </exception>
         public void WriteString(string propertyName, JsonEncodedText value)
-            => WriteString(propertyName.AsSpan(), value);
+            => WriteString((propertyName ?? throw new ArgumentNullException(nameof(propertyName))).AsSpan(), value);
 
         /// <summary>
         /// Writes the property name and string text value (as a JSON string) as part of a name/value pair of a JSON object.
         /// </summary>
         /// <param name="propertyName">The property name of the JSON object to be transcoded and written as UTF-8.</param>
         /// <param name="value">The value to be written as a UTF-8 transcoded JSON string as part of the name/value pair.</param>
         /// <remarks>
+        /// <para>
         /// The property name and value is escaped before writing.
+        /// </para>
+        /// <para>
+        /// If <paramref name="value"/> is <see langword="null"/> the JSON null value is written,
```

Comment:
Reminder, this needs to be manually updated in the docs repo (including other documentation added here):
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.utf8jsonwriter.writestring?view=netcore-3.0#System_Text_Json_Utf8JsonWriter_WriteString_System_String_System_String_

https://github.com/dotnet/dotnet-api-docs/blob/70c0dcdb09b3b8dcb53f0bd6a6f3e2a32ec642eb/xml/System.Text.Json/Utf8JsonWriter.xml#L3639-L3647

Reply:
@carlossanlop - will running your tool capture diffs like this (i.e. additional remarks)?

Reply:
No, the tool will not update API elements that have already been documented. They have to be added manually.

The main reason why the tool should not do diffing of previously documented APIs is because the dotnet-api-docs PRs go through Maira and Ron's language revisions, so in most of the cases the text will not be the same between the triple slash comments and the public documentation.